cmd/podman/pod_start.go

```diff
@@ -0,0 +1,105 @@
+package main
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/pkg/errors"
+	"github.com/projectatomic/libpod/cmd/podman/libpodruntime"
+	"github.com/projectatomic/libpod/libpod"
+	"github.com/urfave/cli"
+)
+
+var (
+	podStartFlags = []cli.Flag{
+		cli.BoolFlag{
+			Name:  "all, a",
+			Usage: "stop all running pods",
+		},
+		LatestFlag,
+	}
+	podStartDescription = `
+   podman pod start
+
+   Starts one or more pods.  The pod name or ID can be used.
+`
+
+	podStartCommand = cli.Command{
+		Name:                   "start",
+		Usage:                  "Start one or more pods",
+		Description:            podStartDescription,
+		Flags:                  podStartFlags,
+		Action:                 podStartCmd,
+		ArgsUsage:              "POD-NAME [POD-NAME ...]",
+		UseShortOptionHandling: true,
+	}
+)
+
+func podStartCmd(c *cli.Context) error {
+	args := c.Args()
+	if (c.Bool("all") || c.Bool("latest")) && len(args) > 0 {
```

Comment:
Can you make a helper function to do this? It seems to come up often... Just passing in the cli.Context and checking for good options seems fine.